lib/klaviyo/apis/profiles.rb

```diff
@@ -32,7 +32,7 @@ def self.get_person_metrics_timeline(person_id, since: nil, count: DEFAULT_COUNT
         :count => count,
         :sort => sort
       }
-      v1_request(HTTP_GET, path, params)
+      v1_request(HTTP_GET, path, CONTENT_JSON, params)
```

Comment:
if content_json is the default argument, do we need to pass it for all of these that don't need it?

Reply:
afaik yes, I have to add the content argument since this instance of calling v1_request passes along 'params' as well.  Line 10 of profiles.rb there is an example of using v1_request without passing in the content since there aren't any params to pass through, I leave out the content_json as well